src/mono/wasm/runtime/loader/globals.ts

```diff
@@ -62,6 +62,7 @@ export function setLoaderGlobals(
 
         maxParallelDownloads: 16,
         enableDownloadRetry: true,
+        assertAfterExit: !ENVIRONMENT_IS_WEB,
```

Comment:
we may want to document this default + config option somewhere (sorry if this is already in the diff elsewhere, commenting as i go)